designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ScenarioCommand.scala

```diff
@@ -10,8 +10,10 @@ import pl.touk.nussknacker.ui.security.api.LoggedUser
 import scala.concurrent.Future
 
 sealed trait ScenarioCommand[Result] {
-  def user: LoggedUser
+  val commonData: CommonCommandData
+}
 
+case class CommonCommandData(processIdWithName: ProcessIdWithName, comment: Option[Comment], user: LoggedUser) {
```

Comment:
I thought about this a while and I think that better would be to group this common fields logically instead of haveing "common data". I mean replacing `commonData` with two fields in the `ScenarioCommand` trait:
- `scenarioIdWithName`
- `auditData: ActivityAuditData`

WDYT?

Reply:
Ignore this, I see that this user is not only for audit purpose, but also for access control.